doc/dev/dataplane_codegen_quick_start.md

```diff
@@ -0,0 +1,198 @@
+
+Here is the [Dataplane Codegen Quick Start for Test](https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-for-Test) about how to write and run tests for the Python SDK.
+
+**Reviewers: We may want to pull the content from this wiki page into here.**
+
+## How to write samples
+
+Samples can make it quicker and easier for customers to use the package. It could be simple which just shows how to use the package; Or it could be a complex scenario example for customers to create their own service quickly.
+
+You should add samples in the `samples` directory under **${PROJECT_ROOT}**. Each sample should have its own folder and contain a `Readme.md`([example](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/webpubsub/azure-messaging-webpubsubservice/samples/Readme.md)) which introduces simply how to run the sample.
+
+If you want to run sample with dev mode, just run `pip install -e .`([detail doc](https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-e)) under **${PROJECT_ROOT}** instead of `pip install azure.myservice`.
+
+## How to create package
+
+If you want to create a private package, go to **${PROJECT_ROOT} folder** and run:
+
+```sh
+python setup.py bdist_wheel
+```
+
+## Create/Update the `ci.yaml`
+
+Now, if everything looks good to you, you can submit a PR in [azure-sdk-for-python](https://github.com/Azure/azure-sdk-for-python) repo with all the changes you made above. Before you do that, you need to add/update the `ci.yml` file. Depends on whether there's already one in `sdk/{service_name}` folder.
+
+If there's no such file then you can add the following template.
+
+```yaml
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+    - restapi*
+  paths:
+    include:
+    - sdk/{service_name}/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+    - restapi*
+  paths:
+    include:
+    - sdk/{service_name}/
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: {service_name}
+    Artifacts:
+    - name: azure-mgmt-webpubsub
+      safeName: azuremgmtwebpubsub
+    - name: azure-messaging-webpubsubservice
+      safeName: azuremessagingwebpubsubservice
+```
+
+Please change the `{service_name}`, `Artifacts name` and `safeName` into yours.
+
+If there's already a `ci.yml` file in your project path. then the only thing you need to do is to add the `Artifacts name` and `safeName` of yours into that `ci.yml`.
+
+Usually, `Artifacts name` is same as `package name` and remove `-`  in `Artifacts name` to get `safeName`. Example: [webpubwub](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/webpubsub/ci.yml), [purview](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/purview/ci.yml)
```

Comment:
I think you also need to run `/prepare-pipelines` or something in your PR if there has been no `ci.yml` / pipelines set up. The PR recommends it too